queries/signing_flow.py

```diff
@@ -170,3 +170,178 @@ def construct_get_signing_flows_by_uuids(ids: List[str]) -> str:
+        }
+    }
+    """)
+    return query_template.substitute(
+        signflow_ids=" ".join(
+            list(map(sparql_escape_string, signflow_ids))
+        ),
+    )
+
+
+def reset_signflows(signflow_ids):
+    query_template = Template("""
+    PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+    PREFIX sign: <http://mu.semte.ch/vocabularies/ext/handtekenen/>
+    PREFIX dct: <http://purl.org/dc/terms/>
+    PREFIX prov: <http://www.w3.org/ns/prov#>
+    PREFIX adms: <http://www.w3.org/ns/adms#>
+    
+    DELETE {
+        ?sign_flow adms:status ?status .
+        ?sign_flow dct:creator ?creator .
+        ?s ?p ?o .
+    } INSERT {
+        ?sign_flow adms:status <http://themis.vlaanderen.be/id/handtekenstatus/f6a60072-0537-11ee-bb35-ee395168dcf7> .
+    } WHERE {
+        VALUES ?id { $signflow_ids }
+
+        ?sign_flow a sign:Handtekenaangelegenheid ;
+            mu:uuid ?id ;
+            adms:status ?status ;
+            dct:creator ?creator ;
+            sign:doorlooptHandtekening ?sign_subcase .
+        ?marking_activity 
+            sign:markeringVindtPlaatsTijdens ?sign_subcase ;
+            sign:gemarkeerdStuk ?piece .
+        {
+            ?piece sign:getekendStukKopie ?s .
+            ?s ?p ?o .
+        } UNION {
+            ?piece sign:getekendStukKopie ?o .
+            BIND(?piece AS ?s)
+            BIND(sign:getekendStukKopie AS ?p)
+        } UNION {
+            ?piece sign:getekendStukKopie ?signed_piece_copy .
+            ?signed_piece_copy prov:value ?s .
+            ?s ?p ?o .
+        } UNION {
+            ?s sign:ongetekendStuk ?piece ; ?p ?o .
+        } UNION {
+            ?signed_piece sign:ongetekendStuk ?piece ; prov:value ?s .
+            ?s ?p ?o .
+        } UNION {
+            ?preparation_activity sign:voorbereidingVindtPlaatsTijdens ?sign_subcase .
+            ?preparation_activity sign:voorbereidingGenereert ?s .
+            ?s ?p ?o .
+        } UNION {
+            ?s sign:voorbereidingVindtPlaatsTijdens ?sign_subcase ; ?p ?o .
+        } UNION {
+            ?s sign:handtekeningVindtPlaatsTijdens ?sign_subcase ; ?p ?o .
+        } UNION {
+            ?s sign:goedkeuringVindtPlaatsTijdens ?sign_subcase ; ?p ?o .
+        } UNION {
+            ?s sign:weigeringVindtPlaatsTijdens ?sign_subcase ; ?p ?o .
+        } UNION {
+            ?s sign:annulatieVindtPlaatsTijdens ?sign_subcase ; ?p ?o .
+        } UNION {
+            ?s sign:afrondingVindtPlaatsTijdens ?sign_subcase ; ?p ?o .
+        }
+    }
+    """)
+    return query_template.substitute(
+        signflow_ids=" ".join(
+            list(map(sparql_escape_string, signflow_ids))
+        ),
+    )
+
+
+def remove_signflows(signflow_ids):
+    query_template = Template("""
+    PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+    PREFIX sign: <http://mu.semte.ch/vocabularies/ext/handtekenen/>
+    PREFIX prov: <http://www.w3.org/ns/prov#>
+
+    DELETE { ?s ?p ?o }
```

Comment:
After this cleanup, the metadata of the physical file still exist:
This is a query using a deleted file URI
```
select * where {
graph ?g {
?s ?p <http://themis.vlaanderen.be/id/bestand/659d5264030b64000e000000> 
}}
```
![image](https://github.com/kanselarij-vlaanderen/digital-signing-service/assets/22245223/d73c603a-bef0-46b8-9565-7198bd2e3a18)


---- 

this is a query with the share file as ?s
```
select * where {
graph ?g {
<share://659d5264030b64000e000001.pdf> ?p ?o .
}}
```

![image](https://github.com/kanselarij-vlaanderen/digital-signing-service/assets/22245223/93203a96-e9ab-418c-a06d-89edfb9a4652)

Reply:
Does this belong here or with the `delete_physical_file` method? makes more sense to put it there? If we remove the file from disk we should remove the metadata in the same breath.
To avoid the DB having the metadata to a file that no longer exists.